client/src/actions/adminDashboardActions/studentByIdAction.js

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+
+export const FETCH_STUDENTBYID_START = 'FETCH_STUDENTBYID_START';
+export const FETCH_STUDENTBYID_SUCCESS = 'FETCH_STUDENTBYID_SUCCESS';
+export const FETCH_STUDENTBYID_FAILURE = 'FETCH_STUDENTBYID_FAILURE';
+
+export const getStudentById = id => dispatch => {
+    dispatch({ type: FETCH_STUDENTBYID_START })
+    // axios.get(`https://speak-out-be-staging.herokuapp.com/api/?table=students&where=id=${id}`)
+    axios.get(`http://localhost:3300/api/?table=students&where=id=${id}`)
```

Comment:
I would actually recommend pulling these two strings out into an environment variable and have them vary based on what environment you're running in.

[Here's an article](https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5) Samantha Egge found for you. 😉